packages/libs/react-ui/src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,100 @@
+import { SystemIcon } from '../Icons';
+
+import {
+  cardTitleClass,
+  colorVariants,
+  containerClass,
+  contentClass,
+  expandClass,
+  footerClass,
+  headerContainerClass,
+  iconContainerClass,
+  iconContainerFullWidthClass,
+  simpleClass,
+  simpleContentClass,
+} from './Notification.css';
+
+import classNames from 'classnames';
+import React, { FC } from 'react';
+
+export interface INotificationProps {
+  icon?: typeof SystemIcon[keyof typeof SystemIcon];
+  title: string;
+  children?: React.ReactNode;
+  displayCloseButton?: boolean;
+  expand?: boolean;
+  color?: 'default' | keyof typeof colorVariants;
+  simple?: boolean;
+}
+
+export const Notification: FC<INotificationProps> = ({
+  icon,
+  title,
+  children,
+  displayCloseButton,
+  color = 'default',
+  simple,
+  expand = false,
+}) => {
+  const isSimple = (simple as boolean) || children === undefined;
+  const Icon = icon || SystemIcon.HelpCircle;
+
+  return (
+    <div
+      className={classNames(
+        [color === 'default' ? containerClass : colorVariants[color!]],
+        {
+          [expandClass]: expand,
+          [simpleClass]: isSimple,
+        },
+      )}
+    >
+      <div className={iconContainerClass}>
+        <Icon size={'md'} />
+      </div>
```

Comment:
Let's take a look together on how we could improve the structure of styling! I think the structure of this component could be simplified by some adjustments